docs/schema/packaging/record_package.md

```diff
@@ -0,0 +1,34 @@
+# Record Package Schema
+
+The record package schema describes the structure of the container for publishing records. The package contains important metadata.
+
+````{admonition} Example
+:class: hint
+
+```{jsoninclude} ../../examples/merging/updates/versioned.json
```

Comment:
The two examples are overwhelmingly not package metadata. I would prefer an example that just puts an empty object where other schemas are involved.

The examples were added to the other pages in #1482 to close #1348. They should be kept there, as these specific examples where intended to demonstrate the record and release (in the context of their package).

Reply:
> The two examples are overwhelmingly not package metadata. I would prefer an example that just puts an empty object where other schemas are involved.

There's no `expand` option specified so the rendered result looks the same and has the advantage of actually being valid data (unlike an empty record object):

![image](https://github.com/open-contracting/standard/assets/19265814/2b5bcbc3-ba06-41d0-8b76-8832f406606b)

Happy to replace it with a minimal example with an empty object, if you still prefer that option.

> The examples were added to the other pages in #1482 to close #1348. They should be kept there, as these specific examples where intended to demonstrate the record and release (in the context of their package).

The sections in which the examples appeared no longer exist. The examples were introduced with a sentence like "Records must be published within a [record package](https://standard.open-contracting.org/staging/1.2-dev/en/schema/record_package/)." Now that packages are optional, I figured it makes sense to provide examples demonstrating the relationship between releases/records and packages on the package reference pages. Where would you want to put the package examples on the release and record reference pages and what framing should we provide?

If we choose to leave them off the release and record pages, I think we should add an example of a record to the record reference page in order to demonstrate what a full release looks like. (The release structure section of the release reference already includes an example of a release.)

Reply:
> There's no expand option specified so the rendered result looks the same

Ah, perfect. That works.

> If we choose to leave them off the release and record pages, I think we should add an example of a record to the record reference page in order to demonstrate what a full record looks like.

Yes, sounds good.

> The sections in which the examples appeared no longer exist.

Oh, right. Your changes sound fine then.

Just a note to myself to take a look at the rendered pages when doing a full review.

Reply:
> > If we choose to leave them off the release and record pages, I think we should add an example of a record to the record reference page in order to demonstrate what a full record looks like.
> 
> Yes, sounds good.

Done in https://github.com/open-contracting/standard/pull/1640/commits/ddfdc0c12406222f14870afe0ebba865b8db02d8